src/app/comments/comment-form-container/comment-form-container.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="comment-form">
+  <h2>Add comment<span> - share your thought</span></h2>
+  <app-comment-form
+    (commentDataToParent)="getCommentDataFromParent($event)"
```

Comment:
`getCommentDataFromParent` gets called when child emit data trough its output, so the name `getCommentDataFromParent` would be more appropriate, but please omit the `fromWhere` naming. and lets call it simplier by its containing data, eg.`comment`, maybe newComment